.github/actions/get-release-issue-v1/src/run.ts

```diff
@@ -0,0 +1,57 @@
+import { getExecOutput } from '@actions/exec'
+import type { GitHub, Core, Issues } from './types'
+
+export default async function run(core: Core, github: GitHub) {
+  try {
+    const version = core.getInput('version', { required: true }).trim()
+    const { owner, repo } = github.context.repo
+    let ownerAndRepo = core.getInput('owner-and-repo').toLowerCase().trim()
+
+    if (!ownerAndRepo) {
+      ownerAndRepo = `${owner}/${repo}`
+    }
+
+    core.info(`Getting issues for ${ownerAndRepo} v${version}...`)
+
+    /**
+     * Note: For release-candidate issues (release-notes) that live in the docs repo. We want to search the repo
+     * for the issue that the release was created for. So we use the `--search` flag to search for
+     * the issue using using the GitHub context.
+     */
+    const isDocsRepo = ownerAndRepo.split('/')[1]?.startsWith('docs-')
+    const ghCommand = isDocsRepo
+      ? `gh issue list --repo ${ownerAndRepo} --label release --state open --json url,title --search "${owner}/${repo} v${version}"`
+      : `gh issue list --repo ${ownerAndRepo} --label release --state open --json url,title`
+
+    const {
+      stdout: issueList,
+      stderr: issuesLIstError,
+      exitCode: issuesListExitCode
+    } = await getExecOutput(ghCommand)
+
+    if (issuesListExitCode) {
+      throw new Error(`Error getting issues: \n${issuesLIstError}`)
+    }
+
+    const issues: Issues = JSON.parse(issueList)
+    const issue = issues.find(
+      /**
+       * Follows the format of: `owner/repo v1.0.0`
+       * @see https://github.com/dequelabs/axe-api-team-public/blob/main/.github/actions/create-release-candidate-v1/action.yml#L170
+       */
+      ({ title }) =>
+        isDocsRepo
+          ? title === `${owner}/${repo} v${version}`
+          : title === `${ownerAndRepo} v${version}`
+    )
+
+    if (!issue) {
+      throw new Error(`No issue found for ${ownerAndRepo} v${version}`)
```

Comment:
Do we want this action to fail if it can't find a ticket? What should happen if we manually close a ticket before running this action? Failing would mean we cannot automatically revert the release since a single ticket isn't found that may not need to be closed.

Reply:
Discussed that we should warn the issue instead of erroring here